src/histolab/slide.py

```diff
@@ -259,9 +259,9 @@ def save_thumbnails(self, n: int = 0) -> None:
     @property
     def slides(self) -> List[Slide]:
         return [
-            Slide(os.path.join(self._slides_path, _path), self._processed_path)
-            for _path in os.listdir(self._slides_path)
-            if os.path.splitext(_path)[1] in self._valid_extensions
+            Slide(os.path.join(self._slides_path, wsi_path), self._processed_path)
```

Comment:
this should be some merge mess ... wsi_path now is _path isn't it? or maybe i'm wrong. In any case check it and correct the code here and eventually the tests

Reply:
yup but here it is just the variable in the for loop

Reply:
i would change the forloop var anyway